packages/ncids-css/packages/nci-header/internal/_navigation-mobile.scss

```diff
@@ -59,18 +59,21 @@
   &.active {
 		display: flex;
   }
-  & .nci-header-mobilenav__list > .nci-header-mobilenav__list-node:first-child:not(.active) {
-    border-top: 1px solid color("gray-warm-10");
-    margin-top: units(6);
-  }
-
 	.nci-is-loading {
 		&:after {
 			margin-top: units(8);
 		}
 	}
 }
 
+.nci-header-mobilenav > .nci-header-mobilenav__list > .nci-header-mobilenav__list-node:first-child:not(.active) {
+	border-top: 1px solid color("gray-warm-10");
+	margin-top: units(6);
+}
+.nci-header-mobilenav > .nci-header-mobilenav__list > .nci-header-mobilenav__list-item:first-child:not(.active) {
```

Comment:
i believe this needs to be `.nci-header-mobilenav__nav > .nci-header-mobilenav__list` per `mobile-menu.ts` generation of the mobile menu

Reply:
updating css and fixing storybook HTML to match